### ISSUE TITLE
in POST v2/subscriptions, forbidden chars are allowed in description field

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2308]
in POST v2/subscriptions, forbidden chars are allowed in description field

#### Dataset (these values are replaced in description value)
```
      | desc        |
      |-------------|
      | house<flat> |
      | house=flat  |
      | house"flat" |
      | house'flat' |
      | house;flat  |
      | house(flat) |
```

#### subscription request
```
url: POST http://qa-orion-fe-02:1026/v2/subscriptions
headers:
    Content-Type: application/json
    Fiware-Service: test_csub_desc
    Fiware-ServicePath: /test
payload: {"notification": {"http": {"url": "http://localhost:1234"}, "attrs": ["temperature"]}, "expires": "2016-04-05T14:00:00.00Z", "description": "house(flat)", "subject": {"entities": [{"idPattern": ".*", "type": "room"}], "condition": {"attrs": ["temperature_0", "temperature_1", "temperature_2"]}}}
```

#### subscription response 
```
http code: 201 - Created
headers:
   date: Mon, 20 Jun 2016 09:41:29 GMT
   fiware-correlator: 2a53de32-36cb-11e6-b0c9-005056a20feb
   connection: Keep-Alive
   content-length: 0
   location: /v2/subscriptions/5767ba49a8c8d9e9f5773a8b
payload:
```

#### expected response 
```
http code: 400 Bad Request 
{
    "error": "BadRequest",     
    "description": "description element has forbidden char"
}
```

#### mongo docs
```
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a86"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house<flat>", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a87"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house=flat", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a88"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house\"flat\"", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a89"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house'flat'", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a8a"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house;flat", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
{ "_id" : ObjectId("5767ba49a8c8d9e9f5773a8b"), "expiration" : NumberLong(1459864800), "reference" : "http://localhost:1234", "custom" : false, "throttling" : NumberLong(0), "servicePath" : "/test", "description" : "house(flat)", "status" : "active", "entities" : [ { "id" : ".*", "type" : "room", "isPattern" : "true" } ], "attrs" : [ "temperature" ], "blacklist" : false, "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature_0", "temperature_1", "temperature_2" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "normalized" }
```

**Comment by fgalan** [comment]
Fixed by PR #2313. Please @iariasleon have a look and close the issue (after asigning to milestone 1.3.0) if you find it ok.

**Comment by iariasleon** [comment]
LGTM
```
http code: 400 - Bad Request
headers:
   date: Fri, 29 Jul 2016 09:24:11 GMT
   fiware-correlator: 358d1df6-556e-11e6-af7d-005056a20feb
   connection: Keep-Alive
   content-type: application/json
   content-length: 74
payload: {"error":"BadRequest","description":"forbidden characters in description"}

```
